tufaceous/src/date.rs

```diff
@@ -0,0 +1,15 @@
+use anyhow::Result;
+use chrono::{DateTime, Duration, Timelike, Utc};
+
+pub(crate) fn parse_duration_or_datetime(s: &str) -> Result<DateTime<Utc>> {
+    match humantime::parse_duration(s) {
+        Ok(duration) => {
+            // Remove nanoseconds from the timestamp to keep it less
+            // overwhelming. `Timelike::with_nanosecond` returns None only when
+            // passed a value over 2 billion
+            let now = Utc::now().with_nanosecond(0).unwrap();
+            Ok(now + Duration::from_std(duration)?)
+        }
+        Err(_) => Ok(s.parse()?),
+    }
+}
```

Comment:
Took me a bit to figure out this is only for parsing CLI args. Not sure if that's worth a comment.